presto-docs/src/main/sphinx/release/release-0.235.rst

```diff
@@ -0,0 +1,64 @@
+* Add optimization for page projection and filter by extract and compute common subexpressions among all projections first. This optimization can be turned off by session property ``optimize_common_sub_expressions``.
+* Add soft memory limit configuration properties. Soft memory limits are default memory limits given to each query that can be overridden using session properties up to the hard limit set by the existing configuration properties. Available soft memory limit configuration properties are ``query.soft-max-memory-per-node``, ``query.soft-max-total-memory-per-node``, ``query.soft-max-total-memory``, and ``query.soft-max-memory``.
+* Add check to disallow invoking SQL functions in SQL function body.
+* Add config property driver.max-page-partitioning-buffer-count, to limit the total number of buffers per optimized repartitioning operator. The default value is 1,000,000.
+* Add peak task total memory to returned stats.
+* Add :func:`scale_qdigest` function to scale a ``qdigest`` to a new weight
+* Add :func:`myanmar_font_encoding` and :func:`myanmar_normalize_unicode` to support working with Burmese text
+* Add configuration property ``hive.orc-compression-codec`` to override ``hive.compression-codec`` for ORC and DWRF formats. If specified, ORC and DWRF files are compressed using this codec. RC, Parquet, and other files are compressed using hive.compression-codec.
+
+Pinot Connector Changes
+_______________________
+* Add support for mapping Pinot ``BYTES`` data type to Presto ``VARBINARY`` type.
+* Add support for mapping Pinot time fields with days since epoch value to Presto ``DATE`` type.
+* Add support for mapping Pinot time fields with milliseconds since epoch value to Presto ``TIMESTAMP`` type.
+* Add Pinot Field type in to column comment field shown as ``DIMENSION``, ``METRIC``, ``TIME``, ``DATETIME``, to provide more information.
+* Add configs ``pinot.infer-date-type-in-schema`` and ``pinot.infer-timestamp-type-in-schema`` to switch on/off the type conversion. Default is OFF.
+
+Verifier Changes
+________________
+* Fix an issue where resubmitted queries always fail.
+* Add support for verifying ``SELECT`` queries that produce columns of ``TIME``, ``TIMESTAMP WITH TIME ZONE``, or ``DECIMAL`` types, or columns of structured types with those types.
+* Add support for specifying table properties override for temporary Verifier tables, through configuration property ``control.table-properties`` and ``test.table-properties``.
+* Add support to output verification results for failures due to Verifier internal errors.
+* Add support to skip teardown queries in case control and test queries succeeds but verification fails. This can be enabled by configuration property ``smart-teardown``, which replaces ``run-teardown-on-result-mismatch``.
+
+Druid Changes
+_____________
+* Fix druid connector segment scan.
+* Add support for query processing pushdown via the ``druid.pushdown`` session property.
```

Comment:
what kinds of things can be pushed down?  if this is a new feature, then we probably don't need the note about count distinct pushdown

Reply:
CC: @zhenxiao this is based on the original:

```
Session property, druid.pushdown, it added, to control whether to pushdown all query processing to druid.
```

Reply:
my bad, should be:
Add session property `druid.compute-pushdown-enabled` to pushdown all computation to druid for Presto Druid connector